JSONRPC/JSONRPC.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.IO;
+using System.Text;
+
+using Newtonsoft.Json.Linq;
+
+public class JSONRPC {
+	private string _endpoint;
+
+	// Basic authentication
+	private string _username;
+	private string _password;
```

Comment:
I'm not a fan of using underscores for private/protected in languages that don't need it (like C#) to begin with.